internal/sync/cache.go

```diff
@@ -322,19 +323,59 @@ func (c *liveStateCache) getSyncedCluster() (clustercache.ClusterCache, error) {
 	return clusterCache, nil
 }
 
-func (c *liveStateCache) Init() {
-	setting := c.loadCacheSettings()
+func (c *liveStateCache) Init() error {
+	setting, err := c.loadCacheSettings()
+	if err != nil {
+		return fmt.Errorf("error on init live state cache: %w", err)
+	}
 	c.cacheSettings = *setting
+	return nil
 }
 
-func (c *liveStateCache) loadCacheSettings() *cacheSettings {
+func (c *liveStateCache) loadCacheSettings() (*cacheSettings, error) {
 	ignoreResourceUpdatesEnabled := false
-	resourcesFilter := &NoopResourceFilter{}
+	globalConfig, err := controllerConfig.GetConfigManagerInstance().GetConfig()
```

Comment:
I guess this is not a field that we will allow to dynamically change without needing to restart the Controller?

Reply:
(I'm okay if we don't deal with that here but we may want to have an Issue for that kind of thing, especially if there are other fields like that.)

Reply:
yeah, right. Dynamic change shouldn't be allowed.

Reply:
Do you figure there would be issues if we enabled that?

Reply:
no, I don't see any issue as this config is only read during cache init. So I think we should be fine.